src/components/votingListViewV2/voteListV2.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import VoteRowV2 from './voteRowV2';
+
+class VoteListV2 extends React.Component {
+  componentWillUpdate(nextProps) {
```

Comment:
componentWillUpdate is deprecated, maybe we can just use the componentDidUpdate here?